src/Views/Form/Templates/Classic/resources/js/form.js

```diff
@@ -248,7 +248,7 @@ function moveDefaultGatewayDataIntoActiveGatewaySection() {
 
     document
         .querySelector('.give-gateway-details')
-        .append(...document.querySelectorAll('#give_purchase_form_wrap fieldset:not(.give-donation-submit)'));
+        .append(...jQuery("#give_purchase_form_wrap").children().not('.give-donation-submit').clone(true, true));
```

Comment:
I try to keep vanilla JS functions free of jQuery and also have control over which elements are copied as it can be beneficial to reduce unnecessary code and avoid introducing unexpected bugs. So a simple modification of the original selector seems to achieve the same result. Can you confirm it?

```suggestion
        .append(...document.querySelectorAll('#give_purchase_form_wrap :is(fieldset:not(.give-donation-submit), script)'));
```

Reply:
Good note, Paulo. We're really trying to move away from jQuery. Honestly, you can do pretty much anything in vanilla JS these days that used to be better done in jQuery, and it's faster.